diff --git a/index.d.ts b/index.d.ts
index 9fa2bcbb4d2549588d074dd69bf656a731df055c..3f2616b88d00dda9692d59534e6ef0934407d085 100755
--- a/index.d.ts
+++ b/index.d.ts
@@ -15,7 +15,7 @@
  *
  * @see https://mochajs.org/api/mocha
  */
-declare class Mocha {
+export class Mocha {
     private _growl;
     private _reporter;
     private _ui;
@@ -306,7 +306,7 @@ declare class Mocha {
     enableGlobalTeardown(enabled: boolean): this;
 }
 
-declare namespace Mocha {
+export namespace Mocha {
     namespace utils {
         /**
          * Compute a slug from the given `str`.
@@ -2601,7 +2601,7 @@ declare namespace Mocha {
  *
  * @see https://mochajs.org/api/global.html#runWithSuite
  */
-declare function run(): void;
+export function run(): void;
 
 /**
  * Execute before running tests.
@@ -2610,7 +2610,7 @@ declare function run(): void;
  *
  * @see https://mochajs.org/api/global.html#before
  */
-declare var before: Mocha.HookFunction;
+export var before: Mocha.HookFunction;
 
 /**
  * Execute before running tests.
@@ -2619,7 +2619,7 @@ declare var before: Mocha.HookFunction;
  *
  * @see https://mochajs.org/api/global.html#before
  */
-declare var suiteSetup: Mocha.HookFunction;
+export var suiteSetup: Mocha.HookFunction;
 
 /**
  * Execute after running tests.
@@ -2628,7 +2628,7 @@ declare var suiteSetup: Mocha.HookFunction;
  *
  * @see https://mochajs.org/api/global.html#after
  */
-declare var after: Mocha.HookFunction;
+export var after: Mocha.HookFunction;
 
 /**
  * Execute after running tests.
@@ -2637,7 +2637,7 @@ declare var after: Mocha.HookFunction;
  *
  * @see https://mochajs.org/api/global.html#after
  */
-declare var suiteTeardown: Mocha.HookFunction;
+export var suiteTeardown: Mocha.HookFunction;
 
 /**
  * Execute before each test case.
@@ -2646,7 +2646,7 @@ declare var suiteTeardown: Mocha.HookFunction;
  *
  * @see https://mochajs.org/api/global.html#beforeEach
  */
-declare var beforeEach: Mocha.HookFunction;
+export var beforeEach: Mocha.HookFunction;
 
 /**
  * Execute before each test case.
@@ -2655,7 +2655,7 @@ declare var beforeEach: Mocha.HookFunction;
  *
  * @see https://mochajs.org/api/global.html#beforeEach
  */
-declare var setup: Mocha.HookFunction;
+export var setup: Mocha.HookFunction;
 
 /**
  * Execute after each test case.
@@ -2664,7 +2664,7 @@ declare var setup: Mocha.HookFunction;
  *
  * @see https://mochajs.org/api/global.html#afterEach
  */
-declare var afterEach: Mocha.HookFunction;
+export var afterEach: Mocha.HookFunction;
 
 /**
  * Execute after each test case.
@@ -2673,77 +2673,77 @@ declare var afterEach: Mocha.HookFunction;
  *
  * @see https://mochajs.org/api/global.html#afterEach
  */
-declare var teardown: Mocha.HookFunction;
+export var teardown: Mocha.HookFunction;
 
 /**
  * Describe a "suite" containing nested suites and tests.
  *
  * - _Only available when invoked via the mocha CLI._
  */
-declare var describe: Mocha.SuiteFunction;
+export var describe: Mocha.SuiteFunction;
 
 /**
  * Describe a "suite" containing nested suites and tests.
  *
  * - _Only available when invoked via the mocha CLI._
  */
-declare var context: Mocha.SuiteFunction;
+export var context: Mocha.SuiteFunction;
 
 /**
  * Describe a "suite" containing nested suites and tests.
  *
  * - _Only available when invoked via the mocha CLI._
  */
-declare var suite: Mocha.SuiteFunction;
+export var suite: Mocha.SuiteFunction;
 
 /**
  * Pending suite.
  *
  * - _Only available when invoked via the mocha CLI._
  */
-declare var xdescribe: Mocha.PendingSuiteFunction;
+export var xdescribe: Mocha.PendingSuiteFunction;
 
 /**
  * Pending suite.
  *
  * - _Only available when invoked via the mocha CLI._
  */
-declare var xcontext: Mocha.PendingSuiteFunction;
+export var xcontext: Mocha.PendingSuiteFunction;
 
 /**
  * Describes a test case.
  *
  * - _Only available when invoked via the mocha CLI._
  */
-declare var it: Mocha.TestFunction;
+export var it: Mocha.TestFunction;
 
 /**
  * Describes a test case.
  *
  * - _Only available when invoked via the mocha CLI._
  */
-declare var specify: Mocha.TestFunction;
+export var specify: Mocha.TestFunction;
 
 /**
  * Describes a test case.
  *
  * - _Only available when invoked via the mocha CLI._
  */
-declare var test: Mocha.TestFunction;
+export var test: Mocha.TestFunction;
 
 /**
  * Describes a pending test case.
  *
  * - _Only available when invoked via the mocha CLI._
  */
-declare var xit: Mocha.PendingTestFunction;
+export var xit: Mocha.PendingTestFunction;
 
 /**
  * Describes a pending test case.
  *
  * - _Only available when invoked via the mocha CLI._
  */
-declare var xspecify: Mocha.PendingTestFunction;
+export var xspecify: Mocha.PendingTestFunction;
 
 // #endregion Test interface augmentations
 
@@ -2780,7 +2780,7 @@ declare namespace NodeJS {
  *
  * - _Only supported in the browser._
  */
-declare const mocha: BrowserMocha;
+export const mocha: BrowserMocha;
 
 interface BrowserMocha extends Mocha {
     /**
